src/main/kotlin/carracing/Car.kt

```diff
@@ -0,0 +1,16 @@
+package carracing
+
+class Car(
+    private var position: Int = 0,
```

Comment:
If you were to use the val keyword, how would you refactor your code with an object?
> What’s the difference, and what benefits does this approach offer?